docs/_pages/releases.md

```diff
@@ -13,6 +13,7 @@ sidebar:
 
 ### Improvements
 * Improve failure message for string assertions when checking for equality - [#2307](https://github.com/fluentassertions/fluentassertions/pull/2307)
+* Allow `IEqualityComparer` in string `BeEquivalentTo` assertion - [#2372](https://github.com/fluentassertions/fluentassertions/pull/2372)
```

Comment:
And `NotBeEquivalentTo`

Reply:
Fixed!
I adapted the PR comment, but totally forgot about the `releases.md` :-)